content/en/tracing/connect_logs_and_traces/dotnet.md

```diff
@@ -215,9 +221,39 @@ using (MappedDiagnosticsLogicalContext.SetScoped("dd.span_id", CorrelationIdenti
 }
 ```
 
+{{% /tab %}}
+{{% tab "Microsoft.Extensions.Logging" %}}
+
+```csharp
+using Datadog.Trace;
+using Microsoft.Extensions.Logging;
+
+ILogger _logger;
+
+// there must be spans started and active before this block.
+using(_logger.BeginScope(new Dictionary<string, object>
+{
+    {"dd.env", CorrelationIdentifier.Env},
+    {"dd.service", CorrelationIdentifier.Service},
+    {"dd.version", CorrelationIdentifier.Version},
+    {"dd.trace_id", CorrelationIdentifier.TraceId.ToString()},
+    {"dd.span_id", CorrelationIdentifier.SpanId.ToString()},
+}))
+{
+    // Log something
+}
+```
+
 {{% /tab %}}
 {{< /tabs >}}
 
+Finally, configure log collection as described in [C# Log Collection][7].
```

Comment:
```suggestion
Lastly, see [C# Log Collection][7] to configure your log collection.
```

Reply:
@andrewlock because this last instruction appears after the example tables, customers may not scroll down completely to see the last step directing them to C# Log Collection. What are your thoughts on moving this sentence to after Step 4?

Reply:
I'm fine with that. The downside is the code to do step 4 would no longer directly follow step 4, but I don't think that's a big problem

Reply:
```suggestion
```